rules/windows/defense_evasion_mshta_beacon.toml

```diff
@@ -1,15 +1,15 @@
 [metadata]
 creation_date = "2020/09/02"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/07/19"
 
 [rule]
 author = ["Elastic"]
 description = """
 Identifies Mshta.exe making outbound network connections. This may indicate adversarial activity as Mshta is often
 leveraged by adversaries to execute malicious scripts and evade detection.
 """
-from = "now-9m"
+from = "now-10m"
```

Comment:
I think we need more margin to account for ingest delays. Having the interval match the sequence max span exactly is tricky

Reply:
```suggestion
from = "now-20m"
```

Reply:
Also updated tests to give 10m buffer baf23b503f9d3004c2f0782ad1a95c7f7eed8c13